src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -231,9 +236,8 @@ public with sharing class PMT_PaymentWizard_CTRL {
     /*******************************************************************************************************
     * @description ActionMethod for the Calculate button that given the current opp outstanding balance, 
     * creates the specified new payments to cover the amount.
-    * @return PageReference null
     */ 
-    public pageReference calculate() {
+    public void calculate() {
```

Comment:
My understanding is that if a page reference is never expected to be return the method should be void and not a null page reference?